addons/project/static/src/views/project_task_kanban/project_task_kanban_model.js

```diff
@@ -42,8 +42,11 @@ export class ProjectTaskKanbanDynamicGroupList extends RelationalModel.DynamicGr
 
 export class ProjectTaskRecord extends RelationalModel.Record {
     _update(changes, options) {
-        const value = changes.personal_stage_type_ids;
-        if (Array.isArray(value)) {
+        let value = changes.personal_stage_type_ids;
+        if (value) {
+            if (!Array.isArray(value)) {
```

Comment:
you can just do the second check I believe, even if the value is false or null the ``Array.isArray()`` should return whether it is an array.

Reply:
Hello @arpr-odoo thank you for your comment 
as you can see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) the case for ``undefined`` input is covered and it should return ``false``
```js
// all following calls return false
...
Array.isArray(undefined);
...
```

Reply:
Yes, I see now you are correct. Thanks for the explanation :100: